Core.Telemetry/OpenTelemetryTrace.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+namespace Microsoft.CloudMine.Core.Telemetry
+{
+    public class OpenTelemetryTrace
+    {
+        public static OpenTelemetryTrace FunctionInvocation = new OpenTelemetryTrace("FunctionInvocation");
+        public static OpenTelemetryTrace ProccessCollectionNode = new OpenTelemetryTrace("ProccessCollectionNode");
+        public static OpenTelemetryTrace Heartbeat = new OpenTelemetryTrace("Heartbeat");
```

Comment:
Would we have issues using the same instance as we had with other static variables?

Also, why not read only?

Reply:
These static variables are only being used to discretize trace naming. Activities are constructed in-context so we should not have trouble using single instances here.